src/owl/config/configure.ml

```diff
@@ -182,6 +182,11 @@ the output of `src/owl/config/configure.exe --verbose`.
 
 let () =
   C.main ~name:"owl" (fun c ->
+      let cblas_conf =
+        let default = { C.Pkg_config.cflags = [] ; libs = ["-cblas"] } in
```

Comment:
I think there you meant `-lcblas` but in this case I would leave that empty, as in most cases we want to use the `cblas` that comes from `openblas` instead (and `-lcblas` used to break some of the builds when I was refactoring the build process and trying to fix ubuntu).

Reply:
Also, I think you should add a test source with `#include <cblas.h> ...` and use that to detect if the `cblas` flags are actually needed.